tooling/airneis/db-populator/category/category.go

```diff
@@ -29,7 +29,7 @@ func CreateCategories(hc httpclient.HttpApi) error {
 		headers := make(map[string]string)
 		headers["content-type"] = "application/json"
 
-		_, status, err := hc.Post("http://localhost:3001/api/categories", headers, bodyToSend)
+		_, status, err := hc.Post("https://back.airneis.orb.local/api/categories", headers, bodyToSend)
```

Comment:
This should be a variable

Reply:
why wasn't it a problem when u first committed it and now it is ?

Reply:
Can you leave this line as it is?. We need to keep it as a localhost as not everyone is on Orbstack. Thanks

Reply:
done